css/mixins.less

```diff
@@ -6,27 +6,15 @@
 
     @font-face {
         font-family: '@{variant}';
-        src: url('@{font-full-path}-@{type}.eot');
-        src: local('@{variant} @{type}'),
-             local('@{variant}-@{type}'),
-             url('@{font-full-path}-@{type}.eot?#iefix') format('embedded-opentype'),
-             url('@{font-full-path}-@{type}.woff2') format('woff2'),
-             url('@{font-full-path}-@{type}.woff') format('woff'),
-             url('@{font-full-path}-@{type}.ttf') format('truetype'),
-             url('@{font-full-path}-@{type}.svg#@{variant}') format('svg');
+        src: url('@{font-full-path}-@{type}.woff2') format('woff2'),
+             url('@{font-full-path}-@{type}.woff') format('woff');
```

Comment:
I guess we still should keep the two `local` variants, so that users, that already have the font on their system just use the local font instead of downloading it.

Reply:
@DanielRuf Ah sorry, did not see your not on the `local` thing. True. Agreed.